ocs_ci/ocs/uninstall.py

```diff
@@ -232,3 +234,100 @@ def uninstall_ocs():
         node_obj.add_label(resource_name=node, label=constants.TOPOLOGY_ROOK_LABEL + '-')
 
     log.info("OCS was removed successfully from cluster ")
+
+
+def uninstall_ocs45():
```

Comment:
we should not have a function specific to a version. Please incorporate the implementation here in the existing uninstall function with conditions for the different versions

Reply:
dummy function just to test the new steps and flow (fully worked :) ) 
will work on merging the two funcs in the next days

Reply:
thanks